kafka/strimzi/README.md

```diff
@@ -1,30 +1,37 @@
 # Kafka Integration
 
+Seldon Core v2 relies on Kafka to implement datacentric inference Pipleines. 
+
+To install kafka in your k8s cluster, we higlight two options, ansible and helm.
+
+
 ## Install Kafka
 
-Clone https://github.com/SeldonIO/ansible-k8s-collection
+### Ansible
 
-Create a kafka operator install in kafka namespace
+We provide automation around the installation of a Kafka cluster for Seldon Core v2 to help with development and testing usecases.
 
-create playbooks/kafka_scv2.yaml
+You can follow the steps defined [here](../../ansible/README.md) to install kafka via ansible.
 
-```
-- name: Install Kafka
-  hosts: localhost
-  roles:
-    - strimzi_kafka
-  vars:
-    strimzi_kafka_create_cluster: false
-    strimzi_kafka_create_metrics: false
-```
+### Helm
+
+Use helm if you require changes and customisations to the Kafka cluster.
+
+The installation of a kafka cluster requires the strimzi operator installed in the same namespace.
+
+One option to install the strimzi operator is via [helm](https://strimzi.io/docs/operators/in-development/full/deploying.html#deploying-cluster-operator-helm-chart-str)
+
+Note that we recommend using KRaft instead of Zookeeper for Kafka. To enable KRaft set `featureGates` during installation via `helm`
 
 ```
-ansible-playbook playbooks/kafka_scv2.yaml
+helm upgrade --install strimzi-kafka-operator strimzi/strimzi-kafka-operator --namespace seldon-mesh  --set featureGates='+UseKRaft\,+UseStrimziPodSets'
 ```
 
-Create our Kafka cluster
+Create Kafka cluster in `seldon-mesh` namespace
```

Comment:
I would highlight here two options:
- (1) using our provided helm chart
- (2) providing minimal `kafka.yaml` that can be applied directly

we could use "tabs" to nicely show the alternatives

Reply:
I am not entirely convinced that we should add here a minimal `kafka.yaml` as  we are describing how to install the Helm chart that we are providing for kafka. Perhaps in some other place  we could add this minimal kafka setup but I am not sure if it adds much given that we have the Chart and users can refer to the more details strimzi  setup that I linked in the docs, IMHO.

Reply:
Would be good to sync with @agrski about what purpose he's seen this Helm chart to serve. 
I see it as nice convenience around customizing Strimzi Kafka install for dev / testing setup. 

I would probably lean towards directing users to use Strimzi directly if they do in-cluster Kafka with guidance from our side limited to what configuration options are relevant to use it with Core V2 pipelines. 

I see value from providing Helm chart and adding some notes how it can be used for creating Kafka cluster but would not want us to be seen as authority on how to configure production Kafka clusters.

Reply:
maybe we can add a note at the top of this file to say it is for SCv2 dev purposes and refer to strimzi docs for details?